source/GponManager/gponmgr_link_state_machine.c

```diff
@@ -177,6 +177,7 @@ ANSC_STATUS gpon_disable_veip_interface(GPON_LINK_SM_CTRL_T* gpon_sm_ctrl)
 #endif
     }
 
```

Comment:
removed the log from gpon_disable_veip_interface api since in the gpon_sm_transition_LinkUp_to_LinkDown logging for both failure and success scenarios.